docs/integrations/containers-orchestration/vmware.md

```diff
@@ -233,7 +233,7 @@ python3 $SCRIPT_PATH/esx_perf_metrics_6_5.py -s 192.168.124.29 -t sumologic_host
   * **-u**: User name to use when connecting to vCenter server. Required.
   * **-p**: Password to use when connecting to vCenter server. Required.
   * **-ts**: Timestamp File. Default ‘.timelog_metrics’.
-  * **-t**: Target Sumologic syslog server. Required.
+  * **-t**: Sumologic installed collector host name for streaming metrics. This is required in case of syslog server.
```

Comment:
For metrics there is no syslog server required. Also in case of metric this is required. We can just write the line : 
"Host name where sumologic installed collector is installed. Required"

Reply:
Fixed this in the latest commit.